Projects/UOContent/Mobiles/Monsters/AOS/AbysmalHorror.cs

```diff
@@ -74,17 +72,9 @@ public override void OnDeath(Container c)
             }
         }
 
-        public override void Serialize(IGenericWriter writer)
+        [AfterDeserialization]
+        private void AfterDeserialization()
```

Comment:
We don't need this, just delete the after deserialization.

Reply:
we can delete it
![image](https://user-images.githubusercontent.com/30377429/208159318-3a1e4aca-1188-41f0-81ac-86ca6aa5791f.png)
inside every mosters code?